models/clipper/ethereum/clipper_ethereum_trades.sql

```diff
@@ -3,14 +3,15 @@
         post_hook='{{ expose_spells(\'["ethereum"]\',
                                 "project",
                                 "clipper",
-                                \'["0xRob"]\') }}'
+                                \'["0xRob", "amalashkevich"]\') }}'
         )
 }}
 
 {% set clipper_models = [
-'clipper_v1_ethereum_trades'
-,'clipper_v2_ethereum_trades'
-,'clipper_v3_ethereum_trades'
+ref('clipper_v1_ethereum_trades')
+, ref('clipper_v2_ethereum_trades')
+, ref('clipper_v3_ethereum_trades')
+, ref'clipper_v4_ethereum_trades')
```

Comment:
Sorry I missed a `(` here:
```suggestion
, ref('clipper_v4_ethereum_trades')
```

Reply:
yep